src/drt/src/io/io.h

```diff
@@ -175,6 +175,8 @@ class Parser
   void genGuides(frNet* net, std::vector<frRect>& rects);
   void genGuides_addCoverGuide(frNet* net, std::vector<frRect>& rects);
   void patchGuides(frNet* net, frBlockObject* pin, std::vector<frRect>& rects);
+  static int distL1(const Rect& b, const Point& p);
+  static void getClosestPoint(const frRect& r, const Point3D& p, Point3D& result);
```

Comment:
Not sure if this is the best way to handle it, but each one is literally used only once in io.cpp